templates/components/panels/production.hbs

```diff
@@ -22,7 +22,7 @@
           <p class="attribution">&ndash; Catherine West, Technical Lead</p>
         </div>
         <div class="four columns">
-          <a href="https://blog.chucklefish.org/about/" target="_blank" rel="noopener"><img src="/static/images/user-logos/chucklefish.png" alt="Chucklefish Logo" /></a>
+          <a href="https://blog.chucklefish.org/about/" target="_blank" rel="noopener"><img src="/static/images/user-logos/chucklefish.png" alt="Chucklefish Logo" id="chucklefish" /></a>
```

Comment:
We could perhaps express this with less code using Tachyons:
```suggestion
          <a href="https://blog.chucklefish.org/about/" target="_blank" rel="noopener"><img src="/static/images/user-logos/chucklefish.png" alt="Chucklefish Logo" class="w-33 w-50-ns" /></a>
```

Reply:
Make complete sense. My apologies, I'll pay more attention to simple tachyon-backed solutions.

Just pushed another commit that removes the class based width style: `width: 80%` and instead gives the images a tachyons `w-` attribute.